services/frontend/src/components/Widgets/Sidebar/AssistantMenuInfo.tsx

```diff
@@ -0,0 +1,140 @@
+import { ReactComponent as Information } from '@assets/icons/information.svg'
+import classNames from 'classnames/bind'
+import { useUIOptions } from 'context'
+import { CSSProperties, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useQuery } from 'react-query'
+import { BotInfoInterface, ISkill, IСounter } from 'types/types'
+import { TOOLTIP_DELAY } from 'constants/constants'
+import { getComponents } from 'api/components'
+import { useAssistants } from 'hooks/api'
+import { useGaDeepy } from 'hooks/googleAnalytics/useGaDeepy'
+import { examination } from 'utils/checkingAssistants'
+import { consts } from 'utils/consts'
+import { trigger } from 'utils/events'
+import { BaseToolTip } from 'components/Menus'
+import { WarningsInfo } from 'components/Panels'
+import { TRIGGER_LEFT_SP_EVENT } from 'components/Panels/BaseSidePanel/BaseSidePanel'
+import { Hint } from 'components/UI'
+import s from './AssistantMenuInfo.module.scss'
+
+///hintIsVisited, setHintIsVisited, HELPER_TAB_ID
+export const HELPER_TAB_ID = 'helperTabError'
+
+export const AssistantMenuInfo = () => {
+  const { fetchPrivateDists } = useAssistants()
+  const privateDistsInit = fetchPrivateDists()
+  const { UIOptions } = useUIOptions()
+  const { deepyChatOpened } = useGaDeepy()
+  const { t } = useTranslation('translation', {
+    keyPrefix: 'assistantMenuInfo',
+  })
+  const copilotIsActive = UIOptions[consts.WARNING_WINDOW_SP_IS_ACTIVE]
+
+  const [privateDists, setPrivateDists] = useState({})
+
+  useEffect(() => {
+    if (privateDistsInit.status === 'success') {
+      setPrivateDists(privateDistsInit)
+    }
+  }, [privateDistsInit.data])
+
+  let cx = classNames.bind(s)
+
+  const handleBtnClick = () => {
+    !copilotIsActive && deepyChatOpened()
+
+    trigger(TRIGGER_LEFT_SP_EVENT as any, {
+      children: <WarningsInfo />,
```

Comment:
```suggestion
    trigger(TRIGGER_LEFT_SP_EVENT, {
```

and extend the TEvents type with the value 'TRIGGER_LEFT_SP_EVENT''